hyp3_metadata/rtc.py

```diff
@@ -0,0 +1,113 @@
+from copy import deepcopy
+from pathlib import Path
+from typing import List, Optional
+
+from osgeo import gdal
+
+from hyp3_metadata.util import get_dem_template_id, get_projection, get_thumbnail_encoded_string, render_template, \
+    strip_polarization
+
+
+class RtcMetadataWriter:
```

Comment:
not sure we actually get much out of this being a class, could just have a create_rtc_metadata_set function that calls the rest of the code.

I do kinda like the encapsulation though

Reply:
these being classes also allows for factory later, allowing the create_meta_data_file file that plugins call to look something like

```py
writer = get_metadata_writer('plugin_name')
writer.write_metadata_file_set(payload)